firefox/manifest.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "OctoLinker",
+  "version": "4.1.1",
```

Comment:
Should we bump the version in chrome/manifest.json as well? I'm assuming they're meant to be synchronized.